docs/guide/concept-di-container.md

```diff
@@ -396,6 +396,18 @@ class HotelController extends Controller
 }
 ``` 
 
+Note that not all core code supports DI due to performance concerns. You can add support if needed by overriding
```

Comment:
I could not find other cases.

I wrote it this way because it seemed the most durable. It is technically correct now and would also be true if a case is discovered or added in the future.

Reply:
I'd move this note into AR guide instead.